src/FormMigration/ServiceProvider.php

```diff
@@ -31,14 +31,16 @@ public function register()
                 Steps\FormTemplate\SequoiaTemplateSettings::class,
                 Steps\FormTemplate\LegacyTemplateSettings::class,
                 Steps\DonationOptions::class,
+                Steps\RecurringDonationOptions::class,
                 Steps\FormFields::class,
                 Steps\FormFields\LoginRegistration::class,
                 Steps\FormFields\CompanyDonations::class,
                 Steps\DonationGoal::class,
                 Steps\TermsAndConditions::class,
                 Steps\FormGrid::class,
                 Steps\OfflineDonations::class,
-                Steps\EmailSettings::class,
+                // TODO: this step was causing errors, so I commented it out
+                //Steps\EmailSettings::class,
```

Comment:
This step was preventing the (migrated) form builder from loading - I think it has something to do with the email settings content not being sanitized for json correctly